packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/vue/next-steps.md

```diff
@@ -1,8 +1,10 @@
-After you complete a basic-sign flow for the Vue.js app, you can extend your app's sign-in experience by adding features such as multifactor, self-registration, or social IdP authentication. For each of these uses cases, you need to thoroughly understand the [Interaction Code flow](/docs/guides/implement-grant-type/interactioncode/main/#interaction-code-flow) to handle all the possible remediation steps of the interaction. In addition to implementing the remediation steps for these use cases in your app code, you need to configure authenticators and authentication policies for your app. See:
+After you create a basic sign-in flow for the Vue.js app, you can extend your app's sign-in experience. You can add features such as multifactor, self-registration, or social IdP authentication.
+
+For each of those features, ensure you understand the [Interaction Code flow](/docs/guides/implement-grant-type/interactioncode/main/#interaction-code-flow) so that you can handle possible remediation steps of the interaction. In addition to implementing the remediation steps for those features in your app code, you need to configure authenticators and authentication policies for your app. See:
```

Comment:
```suggestion
For each of those features, ensure you understand the [Interaction Code flow](/docs/guides/implement-grant-type/interactioncode/main/#interaction-code-flow) so that you can handle possible remediation steps of the interaction. In addition to implementing the remediation steps for those features in your app code, you need to configure authenticators and authentication policies for your app. See these pages for more information:
```